watertap/property_models/NaCl_T_dep_prop_pack.py

```diff
@@ -0,0 +1,1708 @@
+            initialize=dens_mass_C_param_dict,
+            units=pyunits.kg / pyunits.m**3,
+            doc="Mass density parameter C",
+        )
+        self.dens_mass_param_D = Var(
+            dens_mass_D_param_dict.keys(),
+            domain=Reals,
+            initialize=dens_mass_D_param_dict,
+            units=pyunits.kg / pyunits.m**3,
+            doc="Mass density parameter D",
+        )
+        self.dens_mass_param_E = Var(
+            dens_mass_E_param_dict.keys(),
+            domain=Reals,
+            initialize=dens_mass_E_param_dict,
+            units=pyunits.kg / pyunits.m**3,
+            doc="Mass density parameter E",
+        )
+
+        # specific enthalpy parameters, 0-300 C
+        # Sparrow 2003, Eq 8: h = A+BT+CT2+DT3+ET4, where A = ao + a1X + a2X2 + a3X3 + a4X4
+        enth_A_param_dict = {
+            "0": 0.0005e3,
+            "1": 0.0378e3,
+            "2": -0.3682e3,
+            "3": -0.6529e3,
+            "4": 2.89e3,
+        }
+        enth_B_param_dict = {
+            "0": 4.145,
+            "1": -4.973,
+            "2": 4.482,
+            "3": 18.31,
+            "4": -46.41,
+        }
+        enth_C_param_dict = {
+            "0": 0.0007,
+            "1": -0.0059,
+            "2": 0.0854,
+            "3": -0.495,
+            "4": 0.8255,
+        }
+        enth_D_param_dict = {
+            "0": -0.0048e-3,
+            "1": 0.0639e-3,
+            "2": -0.714e-3,
+            "3": 3.273e-3,
+            "4": -4.85e-3,
+        }
+        enth_E_param_dict = {
+            "0": 0.0202e-6,
+            "1": -0.2432e-6,
+            "2": 2.054e-6,
+            "3": -8.211e-6,
+            "4": 11.43e-6,
+        }
+
+        self.enth_param_A = Var(
+            enth_A_param_dict.keys(),
+            domain=Reals,
+            initialize=enth_A_param_dict,
+            units=pyunits.J / pyunits.kg,
+            doc="Specific enthalpy parameter A",
+        )
+        self.enth_param_B = Var(
+            enth_B_param_dict.keys(),
+            domain=Reals,
+            initialize=enth_B_param_dict,
+            units=pyunits.J / pyunits.kg,
+            doc="Specific enthalpy parameter B",
+        )
+        self.enth_param_C = Var(
+            enth_C_param_dict.keys(),
+            domain=Reals,
+            initialize=enth_C_param_dict,
+            units=pyunits.J / pyunits.kg,
+            doc="Specific enthalpy parameter C",
+        )
+        self.enth_param_D = Var(
+            enth_D_param_dict.keys(),
+            domain=Reals,
+            initialize=enth_D_param_dict,
+            units=pyunits.J / pyunits.kg,
+            doc="Specific enthalpy parameter D",
+        )
+        self.enth_param_E = Var(
+            enth_E_param_dict.keys(),
+            domain=Reals,
+            initialize=enth_E_param_dict,
+            units=pyunits.J / pyunits.kg,
+            doc="Specific enthalpy parameter E",
+        )
+
+        # vapor pressure parameters, 0-150 C
```

Comment:
I wonder if this comment should also be above the constraint for pressure_sat and/or in the documentation